### PR TITLE
Add documentation about Swift direct download support in the ER

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: enterprise-registry/swift-temp-url.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+To enable temporary URLs, first set the `X-Account-Meta-Temp-URL-Key` header on your Object Storage account to an arbitrary string. This string serves as a secret key. For example, to set a key of `somecoolkey` using the swift command-line tool:
+
+```
+$ swift post -m "Temp-URL-Key:somecoolkey"
+```
+
+## Visit the Management Panel
+
+Sign in to a super user account and visit `http://yourregister/superuser` to view the management panel:
```
Comment (robszumski) [inline enterprise-registry/swift-temp-url.md]: maybe use `registry.example.com`?

File: enterprise-registry/swift-temp-url.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- Click the configuration tab (<span class="fa fa-gear"></span>) and scroll down to the section entitled <strong>Registry Storage</strong>.
+- Ensure that "OpenStack Storage (Swift)" is selected
+
+## Enter the temporary URL key
+
+Enter the key generated above into the `Temp URL Key` field under the Swift storage engine settings.
+
+## Save configuration
+
+Hit `Save Configuration` to save and validate your configuration. The Swift storage engine system will automatically
```
Comment (robszumski) [inline enterprise-registry/swift-temp-url.md]: erroneous newline here?

Comment (josephschorr) [inline enterprise-registry/swift-temp-url.md]: Doesn't appear in my text. Github is wrapping it

File: enterprise-registry/swift-temp-url.md (hunk 1 above)
Comment (josephschorr) [inline enterprise-registry/swift-temp-url.md]: Done

**Comment by robszumski** [comment]
LGTM

**Pull request merged**